cle/backends/__init__.py

```diff
@@ -528,6 +563,15 @@ def sections(self, v):
         else:
             raise ValueError('Unsupported type %s set as sections.' % type(v))
 
+    def rebase(self):
+        """
+        Rebase backend's regions to the new base where they were mapped by the loader
+        """
+        if self.sections:
+            self.sections._rebase(self.image_base_delta)
```

Comment:
Can we throw an error of some sort if we try to rebase things twice? Currently it'll just keep adding the delta to each of the sections, which is... weird